src/settings/DesktopSettingsViewer.js

```diff
@@ -33,6 +35,7 @@ export class DesktopSettingsViewer implements UpdatableSettingsViewer {
 	_isIntegrated: Stream<?boolean>;
 	_isAutoUpdateEnabled: Stream<?boolean>;
 	_showAutoUpdateOption: Stream<?boolean>;
```

Comment:
I think neither `_showAutoUpdateOption` nor `_updateAvailable` have to be streams